registry/tracecat_registry/templates/detect/lookup_ip_address/ipinfoio.yml

```diff
@@ -0,0 +1,26 @@
+type: action
+definition:
+  name: lookup_ip_address
+  namespace: tools.ipinfoio
+  title: Lookup IP address
+  description: Get IPinfo.io report for an IP address
+  display_group: IPinfo.io

```

Comment:
```suggestion
  display_group: IPinfo

```

Reply:
Follows how Tines calls it https://www.tines.com/solutions/products/ipinfo/. They used to call it ipinfo.io but I think the "." was messing some search up (same for us since "." is a special char).